### PR TITLE
Use packages's gpcloud to test in the main pipeline

### PR DESCRIPTION
**Pull request by adam8157**
Without this commit, test scripts will re-compile gpcloud anyway, better
to use the one from package instead in the main pipeline.

### CODE REVIEW

**Comment by danielgustafsson** [comment]
Forgive my Concourse ignorance, but what does "package" mean in this case? Will the test use gpcloud compiled in a previous step in the pipeline with this? Or can the included gpcloud be older?

Just trying to understand how it all ties together.

**Comment by adam8157** [comment]
@danielgustafsson It's bin_gpdb_centos, gpcloud is in it.

We want to use built-in gpcloud in the main pipeline, and bin_gpdb_centos with latest developing branch's gpcloud in testing pipeline.

File: concourse/scripts/gpcheckcloud_tests_gpcloud.bash [hunk 1]
```diff
@@ -12,8 +12,11 @@ function gen_env(){
 	source /opt/gcc_env.sh
 	source /usr/local/greenplum-db-devel/greenplum_path.sh
 
-	cd "\${1}/gpdb_src/gpAux/extensions/gpcloud"
-	make -B gpcheckcloud
+	if [ "$overwrite_gpcloud" = "true" ]
+	then
+		cd "\${1}/gpdb_src/gpAux/extensions/gpcloud"
```
Comment (cjcjameson) [inline concourse/scripts/gpcheckcloud_tests_gpcloud.bash]: This bash-ism seems overwroght -- can it be simplified?

Comment (adam8157) [inline concourse/scripts/gpcheckcloud_tests_gpcloud.bash]: I think it looks more complex if use `[] &&` to wrap them in oneline.

Comment (cjcjameson) [inline concourse/scripts/gpcheckcloud_tests_gpcloud.bash]: oh sorry, I just meant the "\${ before the 1 -- I think you can get the same effect just with "$1/..."

**Pull request merged**